src/schemas/office-address-schema.ts

```diff
@@ -32,3 +32,5 @@ export const OfficeAddressSchema = {
     maxLength: maxLength(80)
   }
 }
+
+export const RegistrationDeliveryAddressSchema = OfficeAddressSchema
```

Comment:
Would it make sense to move this alias closer to where it's used (as in the component itself)?

If `RegistrationDeliveryAddressSchema` is used in multiple places then, imho, leave this here.

(Ditto for alias in person-address-schema.ts.)

Reply:
Updated
Removed the alias in the schemas themselves and just applied them at the component 👍